notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<p>Adding a little more explanation might make it clearer. </p><blockquote>A function to reindex and reformat time for an event. </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>This fucntion reset the index of the dataframe and reform the dataframe to be used in the get_event function. This get_event needs a new dataframe format </p><p> </p>